tests/test_json.py

```diff
@@ -48,7 +48,7 @@ class MyModel(BaseModel):
 @pytest.mark.parametrize(
     'ser_type,gen_value,json_output',
     [
-        (UUID, lambda: 'ebcdab58-6eb8-46fb-a190-d07a33e9eac8', b'"ebcdab58-6eb8-46fb-a190-d07a33e9eac8"'),
+        (UUID, lambda: UUID('ebcdab58-6eb8-46fb-a190-d07a33e9eac8'), b'"ebcdab58-6eb8-46fb-a190-d07a33e9eac8"'),
```

Comment:
Why was this change necessary?

Reply:
To match the new UUID serializer https://github.com/pydantic/pydantic-core/pull/772/files#diff-3e8029d3656b4c545330496a646d8b651884a649d1a7934fd7f32c50e33fe8a7R10-R20

Do you think it's a regression?